em_workflows/utils/utils.py

```diff
@@ -52,8 +52,8 @@ def lookup_dims(fp: Path) -> Header:
         stderr = sp.stderr.decode("utf-8")
         msg = f"Command ok : {stderr} -- {stdout}"
         log(msg)
-        xyz_dim = re.split(" +(\d+)", stdout)
-        xyz_cleaned = Header(int(xyz_dim[1]), int(xyz_dim[3]), int(xyz_dim[5]))
+        xyz_dim = [int(x) for x in stdout.split()]
+        xyz_cleaned = Header(*xyz_dim)
```

Comment:
Using the unpack operator is perhaps a bit too cute. Maybe this would be better:

`xyz_cleaned = Header(xyz_dim[0], xyz_dim[1], xyz_dim[2])`